go/teams/member_set.go

```diff
@@ -77,27 +77,24 @@ func (m *memberSet) loadMembers(ctx context.Context, g *libkb.GlobalContext, req
 }
 
 func (m *memberSet) loadGroup(ctx context.Context, g *libkb.GlobalContext, group []keybase1.UserVersion, storeRecipient, force bool) ([]member, error) {
-	members := make([]member, len(group))
-	var err error
-	for i, uv := range group {
-		members[i], err = m.loadMember(ctx, g, uv, storeRecipient, force)
+	members := []member{}
```

Comment:
a) `members := []member{}` 
b) `var members []member` 

I'd prefer (b) to (a) because (b) has value `nil` whereas (a) has a nasty non-nil but zero-length value. Not a big deal, but this is a go gotcha I'm on the lookout for. For this same reason I like to do `len(x) == 0` rather than `x == nil` in case someone supplied on of these.